app/src/Manager/AdvisorManager.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Manager;
+
+use App\Entity\Game;
+use Unirest\Request as Request;
+use Unirest\Response as Response;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+class AdvisorManager
+{
+
+    private $adivsorUrl;
+
+    public function __construct(ParameterBagInterface $params)
+    {
+        $this->adivsorUrl = $params->get('advisor_url');
+    }
+
+    private function post($body): Response
+    {
+        $headers = ['Accept' => 'application/json'];
+        return Request::post($this->adivsorUrl.'/best-move', $headers, $body);
+    }
+
+    /**
+     * Return the team of the player stored in the sessoin
```

Comment:
`the team of the player` ? I don't understand ...

Reply:
Yes I don't understand neither... Fix in next PR